cli/node/mod.rs

```diff
@@ -749,6 +749,14 @@ pub fn translate_cjs_to_esm(
     {
       return Ok(analysis);
     }
+
+    if media_type == MediaType::Json {
+      return Ok(CjsAnalysis {
+        exports: vec![],
+        reexports: vec![],
+      });
+    }
```

Comment:
Isn't this just going to make programs expecting JSON data fail at runtime? I'm not sure how aligned to CJS Deno is going to be in compat mode though.

Reply:
How so? I added a test that requires data from JSON file and it works as expected.

Reply:
I see, I misunderstood the code.